docs/reference/commands/tcksift2.rst

```diff
@@ -79,7 +79,7 @@ Options for controlling the SIFT2 optimisation algorithm
 
 -  **-max_coeff_step step** maximum change to a streamline's weighting coefficient in a single iteration (default: 1)
 
--  **-min_cf_decrease frac** minimum decrease in the cost function (as a fraction of the initial value) that must occur each iteration for the algorithm to continue (default: 2.5e-05)
+-  **-min_cf_decrease frac** minimum decrease in the cost function (as a fraction of the initial value) that must occur each iteration for the algorithm to continue (default: 2.5e-005)
```

Comment:
Not really on topic of the pull request, but I've noticed this coming up more often here and there: the precision or number of digits here changing (back and forth, depending on who generated the .rst).  Just wondering: is this something we can control better, or is this dependent on some software version that's out of our hands?